R/mouse/BASE.R

```diff
@@ -0,0 +1,129 @@
+#' Souce code for the BASE algorithm. 
+#' This code is adapted from Varn et al., DOI: 10.1158/0008-5472.CAN-16-2490 
+#' 
+#' @param 
+#---------------------------------------------------------------------------------------------------
+#	data: numeric matrix; Patient gene expression data, with genes as rows and patients as columns
+#	reg: numeric matrix; Reference immune cell weights created using SuppSoftware1
+#	perm: numeric; Number of permutations to perform during the normalization step
+#	myoutf: character; Directory location to save cell lineage score output file to (example: "/home/immune/cell_scores.txt")
+#	median.norm: logical;  parameter stating whether to median center the expression values for each gene
```

Comment:
Could you convert this to a real roxygen docstring, i.e. a `@param` for each parameter?

Reply:
Did it